project-base/app/tests/FrontendApiBundle/Functional/Cart/RemovePromoCodeFromCartTest.php

```diff
@@ -65,20 +49,14 @@ public function applyValidPromoCodeToDefaultCart(): PromoCode
     {
         $promoCode = $this->getReferenceForDomain(PromoCodeDataFixture::VALID_PROMO_CODE, 1, PromoCode::class);
 
-        $applyPromoCodeMutation = 'mutation {
-            ApplyPromoCodeToCart(input: {
-                cartUuid: "' . CartDataFixture::CART_UUID . '"
-                promoCode: "' . $promoCode->getCode() . '"
-            }) {
-                uuid
-                promoCode
-            }
-        }';
-
-        $response = $this->getResponseContentForQuery($applyPromoCodeMutation);
+        $response = $this->getResponseContentForGql(__DIR__ . '/graphql/ApplyPromoCodeToCart.graphql', [
+            'cartUuid' => CartDataFixture::CART_UUID,
+            'promoCode' => $promoCode->getCode(),
+        ]);
         $data = $this->getResponseDataForGraphQlType($response, 'ApplyPromoCodeToCart');
 
-        self::assertEquals($promoCode->getCode(), $data['promoCode']);
+        self::assertEquals($promoCode->getCode(), $data['promoCode']['code']);
+        self::assertEquals($promoCode->getDiscountType(), $data['promoCode']['type']);
```

Comment:
You once use method, that does this and second time you have inline code, pls choose one of the solutions.

Reply:
ah, I must have missed these usages, unified in 18dfe1db88